README.md

```diff
@@ -3,6 +3,49 @@
 El problema a tratar es la dificultad que tienen los agricultores para saber cuál es el precio de compra de cada producto, frutas y verduras, en las diferentes cooperativas agricolas. Cada cooperativa establece diariamente un precio de cada producto que es la cantidad por kilo que pagará al agricultor. Actualmente si se desea conocer el precio debes acudir personalmente a cada una de ellas. El objetivo de la aplicación es comparar los precios que cada cooperativa pone a cada producto y teniendo en cuenta la distancia a la que se encuentra el agricultor de cada cooperativa, calcular a qué cooperativa le conviene ir. Con esta aplicación conseguiriamos ahorrar tiempo y dinero.
 
 Además de esto, si el agricultor decide vender más de un producto a la vez, se creará la ruta óptima la cuál indicará a qué cooperativas tiene que ir y en cual vender cada producto.
+
+## Instalación y uso
+Para usar el proyecto, primero hay que descargar el código. Para ello podemos descargarlo como un zip o clonando el repositorio desde una terminal de esta forma:
+
+```shell
+git clone https://github.com/JaimeGM96/Practica-IV.git
+```
+ 
+Una vez descargado, iremos a la carpeta del proyecto, abriremos una terminal y ejecutaremos el comando:
+
+```shell
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+```
+
+con esto instalaremos Poetry. Podemos instalarlo usando pip, pero de esta manera, nos puede dar problemas de conflictos de dependencias.
+Para comprobar que se ha instalado correctamente, ejecutaremos el comando
```

Comment:
Esto o está copiado del original o está copiado de un compañero. Mal en los dos casos.

Reply:
Lo he puesto a partir de lo que ponía en la documentación oficial

Reply:
Y eso lo has hecho porque...

Reply:
No sabía si daría fallos y no lo he comprobado porque no he probado ese método de instalación

Reply:
¿Podéis entender, por favor, que lo que entregáis es para que yo evalúe lo que habéis aprendido, no cómo sabéis copiar o traducir de otro sitio?

Reply:
Si, fallo mio.